web/middlewares/instance.go

```diff
@@ -46,13 +46,55 @@ func CheckInstanceBlocked(next echo.HandlerFunc) echo.HandlerFunc {
 			return next(c)
 		}
 		if i.CheckInstanceBlocked() {
+			if url, _ := i.ManagerURL(instance.ManagerBlockedURL); url != "" {
+				return c.Redirect(http.StatusFound, url)
```

Comment:
I think it is creating a loop of redirections: when the instance is blocked, the user is redirected from the stack to the manager, but as the user is not logged in on the manager, he is redirected to the login page on the stack, but the instance is blocked, and the user is redirected again to the manager, etc.